src/screens/login-screen.js

```diff
@@ -0,0 +1,99 @@
+import React, {Component} from 'react';
+import {
+  Image,
+  KeyboardAvoidingView,
+  View,
+  StyleSheet,
+  TextInput,
+} from 'react-native';
+import LoginButton from '../components/buttons/login-button';
+import UsersService from '../services/users-service';
+import CurrentUser from '../helpers/current-user';
+import imageLogo from '../assets/images/klep_bot_logo.png';
+
+class LoginScreen extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
```

Comment:
you don't have to destruct this ones, you can easly add 
```
state = {
  a, b
}
```

and remove constructor method.